fast_barcode_scanner_platform_interface/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: fast_barcode_scanner_platform_interface
 description: A common platform interface for the fast_barcode_scanner plugin.
 homepage: https://github.com/jhoogstraat/fast_barcode_scanner/tree/master/fast_barcode_scanner_platform_interface
 repository: https://github.com/jhoogstraat/fast_barcode_scanner
-version: 2.0.0-dev.4
+version: 2.0.6
```

Comment:
Let's call this version 2.1.0. We are changing the interface ins a backwards compatible way.